winpr/libwinpr/path/shell.c

```diff
@@ -264,6 +273,8 @@ char* GetKnownSubPath(int id, const char* path)
 	char* knownPath;
 
 	knownPath = GetKnownPath(id);
+	if (!knownPath)
+		return NULL;
 	subPath = GetCombinedPath(knownPath, path);
 
```

Comment:
@nfedera this isn't necessary in this case since if GetCombinedPath fails it returns NULL and we would free(knownPath) and return NULL. So no special need to handle the error case.